src/official-widgets/merchandise-search-bar/default-config.ts

```diff
@@ -0,0 +1,387 @@
+    viewAllProducts: 'ดูสินค้าทั้งหมด',
+    popularChoices: 'ตัวเลือกยอดนิยม',
+    trending: 'กำลังมาแรง',
+    errorMessage: 'คุณได้ป้อนคำค้นหาหรือรูปภาพที่ไม่ถูกต้อง โปรดลองอีกครั้ง',
+  },
+};
+
+export const DEFAULT_CUSTOMIZATIONS: WidgetConfig['customizations'] = {
+  breakpoints: {
+    mobile: {
+      maxWidth: 767,
+    },
+    tablet: {
+      maxWidth: 1023,
+    },
+  },
+  productGrid: {
+    mobile: {
+      productsPerRow: 2,
+      marginVertical: 16,
+      marginHorizontal: 8,
+    },
+    tablet: {
+      productsPerRow: 3,
+      marginVertical: 16,
+      marginHorizontal: 8,
+    },
+    desktop: {
+      productsPerRow: 4,
+      marginVertical: 16,
+      marginHorizontal: 8,
+    },
+  },
+  productCard: {
+    imageAspectRatio: '2 / 3',
+    openLinksInNewTab: false,
+    price: {
+      show: true,
+      font: {
+        mobile: {
+          size: 12,
+          weight: 400,
+        },
+        tablet: {
+          size: 12,
+          weight: 400,
+        },
+        desktop: {
+          size: 14,
+          weight: 400,
+        },
+      },
+      fontColor: '#EF4444',
+      fontColorDark: '#EF4444',
+    },
+    originalPrice: {
+      show: true,
+      font: {
+        mobile: {
+          size: 12,
+          weight: 400,
+        },
+        tablet: {
+          size: 12,
+          weight: 400,
+        },
+        desktop: {
+          size: 14,
+          weight: 400,
+        },
+      },
+      fontColor: '#9CA3AF',
+      fontColorDark: '#9CA3AF',
+    },
+    title: {
+      show: true,
+      fieldSource: 'title',
+      font: {
+        mobile: {
+          size: 14,
+          weight: 700,
+        },
+        tablet: {
+          size: 14,
+          weight: 700,
+        },
+        desktop: {
+          size: 16,
+          weight: 700,
+        },
+      },
+    },
+    secondaryTitle: {
+      show: false,
+      fieldSource: 'brand',
+      font: {
+        mobile: {
+          size: 12,
+          weight: 400,
+        },
+        tablet: {
+          size: 12,
+          weight: 400,
+        },
+        desktop: {
+          size: 14,
+          weight: 400,
+        },
+      },
+    },
+  },
+  imageUpload: {
+    enable: true,
+    icon: {
+      color: '#929292',
+      colorDark: '#929292',
+    },
+    images: [
+      {
+        url: 'https://cdn.visenze.com/images/widget-1.jpg',
+        label: '',
+      },
+      {
+        url: 'https://cdn.visenze.com/images/widget-2.jpg',
+        label: '',
+      },
+      {
+        url: 'https://cdn.visenze.com/images/widget-3.jpg',
+        label: '',
+      },
+      {
+        url: 'https://cdn.visenze.com/images/widget-4.jpg',
+        label: '',
+      },
+      {
+        url: 'https://cdn.visenze.com/images/widget-5.jpg',
+        label: '',
+      },
+    ],
+  },
+  popularTerms: {
+    enable: true,
+    terms: ['dress', 'formal shirts', 'shoes'],
+  },
+  trendingProducts: {
+    enable: true,
+    products: [
```

Comment:
This should be empty by default, because al the contents in here are expected to be very, very catalog-specific.